Building to iOS fails: Undefined symbols errors regarding CesiumGltfReader

When building Unity project with XCode for iOS it gives errors regarding the CesiumGltfReader. 

`Undefined symbols for architecture arm64:
  "_meshopt_decodeFilterExp", referenced from:
      CesiumGltfReader::decodeMeshOpt(CesiumGltf::Model&, CesiumGltfReader::GltfReaderResult&) in libCesiumGltfReader.a(decodeMeshOpt.o)
  "_meshopt_decodeFilterOct", referenced from:
      CesiumGltfReader::decodeMeshOpt(CesiumGltf::Model&, CesiumGltfReader::GltfReaderResult&) in libCesiumGltfReader.a(decodeMeshOpt.o)
  "_meshopt_decodeFilterQuat", referenced from:
      CesiumGltfReader::decodeMeshOpt(CesiumGltf::Model&, CesiumGltfReader::GltfReaderResult&) in libCesiumGltfReader.a(decodeMeshOpt.o)
  "_meshopt_decodeIndexBuffer", referenced from:
      CesiumGltfReader::decodeMeshOpt(CesiumGltf::Model&, CesiumGltfReader::GltfReaderResult&) in libCesiumGltfReader.a(decodeMeshOpt.o)
  "_meshopt_decodeIndexSequence", referenced from:
      CesiumGltfReader::decodeMeshOpt(CesiumGltf::Model&, CesiumGltfReader::GltfReaderResult&) in libCesiumGltfReader.a(decodeMeshOpt.o)
  "_meshopt_decodeVertexBuffer", referenced from:
      CesiumGltfReader::decodeMeshOpt(CesiumGltf::Model&, CesiumGltfReader::GltfReaderResult&) in libCesiumGltfReader.a(decodeMeshOpt.o)
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)`

Unity version: 2021.3.27
Cesium version: 1.5.0
Xcode version: 14.2

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,5 +1,11 @@
 # Change Log
 
+### ? - ?
+
+##### Fixes :wrench:
+
+- Fixed a bug that prevented building on iOS.
+
 ### v1.5.0 - 2023-08-01
 
 ##### Fixes :wrench:

--- a/native~/CMakeLists.txt
+++ b/native~/CMakeLists.txt
@@ -78,6 +78,7 @@ if (${CMAKE_SYSTEM_NAME} STREQUAL "iOS")
         GSL
         httplib
         ktx_read
+        meshoptimizer
         modp_b64
         spdlog
         sqlite3
@@ -92,6 +93,7 @@ if (${CMAKE_SYSTEM_NAME} STREQUAL "iOS")
         webputilsdecode
         zlibstatic
     )
+    set(MESHOPT_BUILD_SHARED_LIBS OFF BOOL FORCE)
     install(TARGETS tidy-static)
     install(TARGETS ${ALL_TARGETS} PUBLIC_HEADER EXCLUDE_FROM_ALL)
     set_target_properties(cesium-native-tests PROPERTIES EXCLUDE_FROM_ALL 1 EXCLUDE_FROM_DEFAULT_BUILD 1)

